selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/account/TestTestKeycloakPasswordPagePage.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2012-2018 Red Hat, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.selenium.pageobject.dashboard.account;
+
+import static java.util.Arrays.asList;
+import static org.eclipse.che.selenium.pageobject.dashboard.account.TestTestKeycloakPasswordPagePage.PasswordLocators.ERROR_ALERT;
+import static org.eclipse.che.selenium.pageobject.dashboard.account.TestTestKeycloakPasswordPagePage.PasswordLocators.NEW_PASSWORD_CONFIRMATION_ID;
+import static org.eclipse.che.selenium.pageobject.dashboard.account.TestTestKeycloakPasswordPagePage.PasswordLocators.NEW_PASSWORD_FIELD_ID;
+import static org.eclipse.che.selenium.pageobject.dashboard.account.TestTestKeycloakPasswordPagePage.PasswordLocators.PASSWORD_FIELD_ID;
+import static org.eclipse.che.selenium.pageobject.dashboard.account.TestTestKeycloakPasswordPagePage.PasswordLocators.SAVE_BUTTON;
+import static org.eclipse.che.selenium.pageobject.dashboard.account.TestTestKeycloakPasswordPagePage.PasswordLocators.SUCCESS_ALERT;
+import static org.openqa.selenium.support.ui.ExpectedConditions.visibilityOfElementLocated;
+
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+import org.eclipse.che.selenium.core.SeleniumWebDriver;
+import org.openqa.selenium.By;
+import org.openqa.selenium.support.ui.ExpectedCondition;
+
+@Singleton
+public class TestTestKeycloakPasswordPagePage extends TestKeycloakAbstractPage {
```

Comment:
Name looks strange. It's better don't start page object class name from "Test" prefix to not to confuse with selenium tests classes.